cocos/core/director.ts

```diff
@@ -762,9 +762,7 @@ export declare namespace Director {
     export type OnLoadSceneProgress = (completedCount: number, totalCount: number, item: any) => void;
 }
 
-legacyCC.Director = Director;
-
 /**
  * 导演类。
  */
-export const director: Director = Director.instance = legacyCC.director = new Director();
```

Comment:
This shouldn't be changed, it's for backward compatibility

Reply:
already fallbacked and tested